source/geometries/Next100SiPM.cc

```diff
@@ -33,6 +33,8 @@ Next100SiPM::Next100SiPM():
   mother_depth_       (0),
   naming_order_       (0),
   time_binning_       (1.0 * us),
+  with_coating_       (false),
+  coating_thickn_     (2. * micrometer),
```

Comment:
To me the `with_coating` variable is redundant since `coating_thickn_` could mean no coating

Reply:
I could remove it, and leave the SiPMs coated by default unless `coating_thickn_` is defined as 0.